tasks/sep/019-fp-holocene-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,24 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
+
```

Comment:
We also need to call out the:
- The nonce increments that happen
- The approvedHashes mapping being set

Similar to overrides, check out other validation files for a template. The nonce increment can be a bullet point explaining each, the approved hashes is a bit more involved as you can see

Reply:
Agreed with @mds1 that this will only be addressed in the follow up task to fix the Sepolia FPs with the correct `DelayedWETH` implementation.